WindowsServerDocs/get-started/extended-security-updates.md

```diff
@@ -41,6 +41,9 @@ For more information, see the [Extended Security Updates frequently asked questi
 
 If you run Windows Server 2008 or 2008 R2 VMs in Azure, they're automatically enabled for Extended Security Updates. You don't need to configure anything, and there's no additional charge for using Extended Security Updates with Azure VMs. Extended Security Updates are automatically delivered to Azure VMs if they're configured to receive updates.
 
+> [!NOTE]
+> Microsoft.ClassicCompute type VMs require additional configuration for Extended Security Updates deployment, as they don't have access to the Azure Resource Manager, in particular the [Azure Instance Metadata Service](https://docs.microsoft.com/azure/virtual-machines/windows/instance-metadata-service), which is used for determining ESU eligibility. Please contact [Microsoft support](http://support.microsoft.com/contactus?PID=17336) for more help.
```

Comment:
@andreiztm  After looking at this a bit more, when we spell out ESU it gets a bit long. I would suggest chopping it down slightly. What do you think about: 
"Microsoft.ClassicCompute VMs require additional configuration for Extended Security Updates deployment, as they don't have access to the [Azure Instance Metadata Service](https://docs.microsoft.com/azure/virtual-machines/windows/instance-metadata-service), which determines your Extended Security Updates eligibility. Please contact [Microsoft support](http://support.microsoft.com/contactus?PID=17336) for more help."